input/fsh/profiles/FinnishAppointmentPractitioner.fsh

```diff
@@ -0,0 +1,30 @@
+Profile: FinnishAppointmentPractitioner
```

Comment:
There's probably nothing scheduling specific here?
We should change to use the FinnishPractitioner from the base profiles.